### PR TITLE
FurnaceManager feature implemented

### PR DESCRIPTION
**Pull request by SaAlexis**

### CODE REVIEW

File: Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml [hunk 1]
```diff
@@ -41,34 +63,34 @@
...
                                 </Grid>
                             </StackPanel>
                         </DataTemplate>
                     </ListBox.ItemTemplate>
                 </ListBox>
-            </WrapPanel>
+            </StackPanel>
 
-            <StackPanel Orientation="Horizontal" DockPanel.Dock="Top" Height="100" Margin="0,0,0,0">
+            <!--<StackPanel Orientation="Horizontal" DockPanel.Dock="Top" Height="100" Margin="0,0,0,0">
```
Comment (JohanRISPAL) [inline Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml]: Pourquoi il y a un commentaire ici ?

File: Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml [hunk 2]
```diff
@@ -80,26 +102,60 @@
...
-                    </Viewbox>
-                    <Viewbox Grid.Row="7" Height="25" HorizontalAlignment="Left" VerticalAlignment="Center">
-                        <RadioButton x:Name="rbNull" GroupName="GrItem" FontSize="14" HorizontalAlignment="Left" VerticalAlignment="Top" HorizontalContentAlignment="Left" VerticalContentAlignment="Center" Padding="5,-4,0,0" Foreground="White" >Nothing</RadioButton>
-                    </Viewbox>
+                    <StackPanel>
+                        <Label Grid.Row="1" Foreground="White" FontSize="20">Name</Label>
+                        <TextBox x:Name="tbNameItem" Grid.Row="2" DataContext="{Binding ItemsList.CurrentItem}" Text="{Binding Name}" Margin="0,0,0,10" FontSize="20" Height="35" HorizontalContentAlignment="Left" VerticalContentAlignment="Center" />
+                        <!--, UpdateSourceTrigger=PropertyChanged-->
```
Comment (JohanRISPAL) [inline Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml]: Pareil ici

File: Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml (hunk 1 above)
Comment (SaAlexis) [reply Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml]: Resolved 9cd3ac9663e07ec8a027369314a612e5576f052f

File: Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml (hunk 2 above)
Comment (SaAlexis) [reply Server/Mine2CraftWinApp/UserControls/ItemManagerPage.xaml]: Resolved 9cd3ac9663e07ec8a027369314a612e5576f052f

**Pull request merged**